framework/src/userobjects/TimedElementSubdomainModifier.C

```diff
@@ -0,0 +1,83 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// Base class to move elements to a specific subdomain at the given times.
+
+#include "TimedElementSubdomainModifier.h"
+
+registerMooseObject("MooseApp", TimedElementSubdomainModifier);
+
+InputParameters
+TimedElementSubdomainModifier::validParams()
+{
+  InputParameters params = ElementSubdomainModifier::validParams();
+  params.addParam<std::vector<Real>>("times", "The times of the subdomain modifications.");
+
+  return params;
+}
+
+TimedElementSubdomainModifier::TimedElementSubdomainModifier(const InputParameters & parameters)
+  : ElementSubdomainModifier(parameters)
+{
+}
+
+void
+TimedElementSubdomainModifier::initialize()
+{
+  // state variables
+  _last_t = -1;
+  _current_step = _t_step - 1;
```

Comment:
These members need to be declared restartable.

Reply:
Can you please describe in more detail what needs to be done here?